_includes/commands/imagetools-cmds.md

```diff
@@ -0,0 +1,18 @@
+
+| Command                | Description                                          |
+| ---------------------- | ---------------------------------------------------- |
+| mceEditImage           | Opens the edit image dialog.                         |
```

Comment:
This could be confused with the image dialog, so maybe clarify that it's the imagetools dialog? eg: `Opens the imagetools editing dialog.`

Reply:
done